install_scripts/getSimId.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var exec = require('child_process').exec;
+var path = require('path');
+
+var SP = require('serialport');
+var SerialPort = SP.SerialPort;
+
+var privatePath = path.join(__dirname, '..', 'PRIVATE.json');
+
+exec('cat ' + privatePath + ' | grep -e \'"sim": "undefined"\'', function (err, stdout) {
```

Comment:
Instead of the `path.join`+ `exec('cat '+path` dance, you can just do : 
`var private = require("../PRIVATE.json")` and then `if(private.sim === null)` (or whatever the relevant test is)